renderer-macos/lib/stubs/Cocoa_helpers_stubs.h

```diff
@@ -0,0 +1,3 @@
+#import <Cocoa/Cocoa.h>
+
+NSColor *ml_NSColor_make(double red, double green, double blue, double alpha);
```

Comment:
I think we should minimise the amount of .h files. It's just a hassle to maintain.